practice-project-ui/.env

```diff
@@ -0,0 +1 @@
+BASE_URL = "http://localhost:8000"
```

Comment:
I should have mentioned that you should put `.env` file in the `.gitignore` as well

Reply:
Also newline